### DISCUSSION TITLE
Next Button

### DISCUSSION THREAD

**Discussion by zelalemgetahun9374** [OpenMined/courses#216] [Course: Our Privacy Opportunity] [answered]
Hi, currently I am in lesson 3 concept 3. But in every lesson I have completed, the next button almost never worked smoothly. I had to reload the page and click the next button 2 or 3 times. I repeat this step until it works. I just wanted to ask if anyone had faced this challenge.

**Comment by tanmaylaud** [comment, marked as answer]
@zelalemgetahun9374 which browser and device are you working on ? 
I used Chrome on MacOS, did not face any issue. 
However, when I tried chrome on mobile ( in desktop mode) , I did face a similar issue.

**Reply by zelalemgetahun9374** [reply]
@tanmaylaud I used yandex browser version 20.12.1.178 on windows 10. One thing that I noticed during many trials after posting this comment is that  the button works without reloading if I scrolled the page to the end and the up to the beginning and press the next button and wait for around 5 seconds.

**Reply by tanmaylaud** [reply]
@zelalemgetahun9374 in that case, I would suggest using the Chrome/Firefox browser. Probably something to do with browser incompatibility.  Hope that helps :)

**Reply by Nilanshrajput** [reply]
@zelalemgetahun9374  let us know if your problem is solved!

**Reply by hwrdtm** [reply]
@zelalemgetahun9374 bumping once more! Please let us know if you'd need more assistance on this issue, otherwise it'd be great if you can mark this question as answered 🙏 .

**Comment by znreza** [comment]
Just adding to @tanmaylaud , I tried with Safari on Mac and it works fine as well. Maybe some compatibility issue with yandex browser.

**Comment by lahmuller** [comment]
I have the same problem with Chrome on win10.
